internal/app/orchestrator/orchestrator.go

```diff
@@ -0,0 +1,31 @@
+package orchestrator
+
+import (
+	gcp "github.com/envoyproxy/go-control-plane/pkg/cache"
+)
+
+// Orchestrator provides an interface that handles the discovery requests from sidecars.
```

Comment:
```suggestion
// Orchestrator provides an interface that handles the discovery requests from xDS clients.
```

xDS clients being the downstreams and origin servers being the upstream.